src/components/Header/Header.js

```diff
@@ -54,6 +56,14 @@ class Header extends Component {
     this.modalToggle();
   };
 
+  btnColor = () => {
+    if (this.props.savedResource.length > 1) {
```

Comment:
this will be true when there are 2 or more `savedResource`s. Could you change to work for 1 or more?

Reply:
Sure, I can do that.